thing-url-adapter.js

```diff
@@ -726,7 +729,7 @@ function loadThingURLAdapter(addonManager) {
     for (const url of config.urls) {
       adapter.loadThing(url);
     }
-
+    adapter.pollInterval = config.pollInterval * 1000;
```

Comment:
```js
if (typeof config.pollInterval === 'number') {
  adapter.pollInterval = config.pollInterval * 1000;
}
```

Also, this needs to be set _before_ any calls to `adapter.loadThing()`, so before the `for` loop above.

Reply:
> ```js
> if (typeof config.pollInterval === 'number') {
>   adapter.pollInterval = config.pollInterval * 1000;
> }
> ```
> 
> Also, this needs to be set _before_ any calls to `adapter.loadThing()`, so before the `for` loop above.

i'm just curious if pollInteval is defined in configuration as number, is it possible to get not number here? Does config loading/saving check types?

Reply:
The primary case I want to protect against here is for users upgrading their add-on. In that case, this config option would not be set at all, so the type would be `undefined`.